blockstack/stacks-wallet
# Software wallet

This enables creation and restoring of software-only wallets and supports transaction signing using the seed phrase.

Steps to test:
1. Go through onboarding and choose the software option
2. Send a transaction and input the seed phrase written down from onboarding
3. Verify that transaction was successfully broadcasted and subsequently confirmed via the explorer.
